src/foundry/client/pixi/placeables/token.d.ts

```diff
@@ -164,30 +210,35 @@ declare global {
     get emitsLight(): boolean;
 
     /**
-     * Test whether the Token has a limited angle of vision or light emission which would require sight to update on Token rotation
+     * Test whether the Token uses a limited angle of vision or light emission.
      */
-    get hasLimitedVisionAngle(): boolean;
+    get hasLimitedSourceAngle(): boolean;
 
     /**
-     * Translate the token's sight distance in units into a radius in pixels.
-     * @returns The sight radius in pixels
+     * Translate the token's dim light distance in units into a radius in pixels.
      */
     get dimRadius(): number;
 
     /**
      * Translate the token's bright light distance in units into a radius in pixels.
-     * @returns The bright radius in pixels
      */
     get brightRadius(): number;
 
     /**
-     * The named identified for the source object associated with this Token
+     * Translate the token's vision range in units into a radius in pixels.
+     */
+    get sightRange(): number;
+
+    /**
+     * Translate the token's maximum vision range that takes into account lights.
      */
-    get sourceId(): string;
+    get optimalSightRange(): number;
+
+    override clone(): ThisType<Token>;
 
     /**
      * Update the light and vision source objects associated with this Token
-     * @param options - (default: `{}}`)
+     * @param options - Options which configure how perception sources are updated
```

Comment:
Is there no longer a default?

Reply:
There is the `{}`, is that important to document still?

Reply:
It's not critically important but may as well.